scripts/rewrite/ralph-rewrite-loop.sh

```
@@ -121,17 +134,21 @@ phase_address_feedback() {
 
     # If no new comments since our last push, feedback was resolved.
     local now_count; now_count=$(count_cr_pr_comments "$pr_num")
     if [ "$round" -gt 1 ] && [ "$now_count" -eq "$prev_count" ]; then
       log "  no new CR comments after push — treating as resolved"
+      PR_NUM="$pr_num" emit_event "B-review" "passed" "no new feedback" "" "{\"round\":$round}"
       return 0
     fi
 
     log "  round ${round}/${CR_MAX_ROUNDS} — addressing CR feedback"
+    PR_NUM="$pr_num" emit_event "B-review" "started" "" "" "{\"round\":$round}"
+    local round_start; round_start=$(date +%s)
```

Shellcheck (0.11.0)

[warning] round_start appears unused. Verify use (or export if used externally).

(SC2034)